### ISSUE TITLE
MultiInputForm++ ListView: Broken pattern in background colour

### ISSUE THREAD

**Issue by andydandy74** [MostafaElAyoubi/Data-shapes#14]
Hi Mostafa,
I just cam across a ListView where some adjacent rows had the same background colour.
Haven't investigated yet but will post more info if I encounter it in other contexts as well.
![backgroundcolorlistviewbug](https://cloud.githubusercontent.com/assets/3014437/25810280/ef0b0b6e-340f-11e7-90e0-6c38d84bea67.PNG)

**Comment by MostafaElAyoubi** [comment]
Hi Adreas, 
I couldn't manage to reproduce this issue. Please let me know if you can provide a dummy example , I'd like to look Into it :)

**Comment by andydandy74** [comment]
Okay, I'll try to create a minimal reproducible case and post it here.

**Comment by andydandy74** [comment]
@MostafaElAyoubi - I came across this today again by pure chance. 
Here's a simple case that I can consistently reproduce on my machine using the latest code on the repo:
https://www.dropbox.com/s/s7mofhv8j9m6l82/DataShapesListViewIssue.ZIP?dl=0
Let me know once you have downloaded the file so I can delete it.

**Comment by MostafaElAyoubi** [comment]
Great thanks, I downloaded the files

**Comment by MostafaElAyoubi** [comment]
@andydandy74  ok that was very valuable info :) It turns out I was using two different sorting methods: the method built in the ListView class and the default python sort() method. Both seem to not give the same result, which in some cases messed the color pattern. I solved it by sorting the items once and for all with the built in python method. I also seized the opportunity to make the sorting optional.
I'm updating the package right now.
